<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="2" total="7" passed="2">
  <reporter-output>
    <line>
      <![CDATA[Trying to start the browser and getting application ready]]>
    </line>
    <line>
      <![CDATA[Browser and application is up and running fine]]>
    </line>
    <line>
      <![CDATA[Trying to start the browser and getting application ready]]>
    </line>
    <line>
      <![CDATA[Browser and application is up and running fine]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="118987" started-at="2019-11-22T21:21:50Z" finished-at="2019-11-22T21:23:49Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="118987" started-at="2019-11-22T21:21:50Z" finished-at="2019-11-22T21:23:49Z">
      <class name="com.learnautomation.testCases.LoginTest">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="setUpSuite" is-config="true" duration-ms="2226" started-at="2019-11-22T21:21:48Z" finished-at="2019-11-22T21:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="setUp" is-config="true" duration-ms="14807" started-at="2019-11-22T21:21:51Z" finished-at="2019-11-22T21:22:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://mail.integramicro.co.in/bizmail/login]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Trying to start the browser and getting application ready]]>
            </line>
            <line>
              <![CDATA[Browser and application is up and running fine]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="login" duration-ms="9118" started-at="2019-11-22T21:22:05Z" finished-at="2019-11-22T21:22:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.learnautomation.testCases.LoginTest.login(LoginTest.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="tearDownMethod" is-config="true" duration-ms="52" started-at="2019-11-22T21:22:14Z" finished-at="2019-11-22T21:22:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=SKIP method=LoginTest.login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="SKIP" signature="login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="login" duration-ms="34148" started-at="2019-11-22T21:22:15Z" finished-at="2019-11-22T21:22:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.learnautomation.testCases.LoginTest.login(LoginTest.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="tearDownMethod" is-config="true" duration-ms="8" started-at="2019-11-22T21:22:49Z" finished-at="2019-11-22T21:22:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=SKIP method=LoginTest.login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="FAIL" signature="login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="login" duration-ms="34034" started-at="2019-11-22T21:22:49Z" finished-at="2019-11-22T21:23:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.learnautomation.testCases.LoginTest.login(LoginTest.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="tearDownMethod" is-config="true" duration-ms="727" started-at="2019-11-22T21:23:23Z" finished-at="2019-11-22T21:23:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=LoginTest.login()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="tearDown" is-config="true" duration-ms="1107" started-at="2019-11-22T21:23:23Z" finished-at="2019-11-22T21:23:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownSuite()[pri:0, instance:com.learnautomation.testCases.LoginTest@7d9d1a19]" name="tearDownSuite" is-config="true" duration-ms="1245" started-at="2019-11-22T21:23:50Z" finished-at="2019-11-22T21:23:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSuite -->
      </class> <!-- com.learnautomation.testCases.LoginTest -->
      <class name="com.learnautomation.testCases.LoginLogoutTest">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="setUp" is-config="true" duration-ms="11877" started-at="2019-11-22T21:23:25Z" finished-at="2019-11-22T21:23:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://mail.integramicro.co.in/bizmail/login]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Trying to start the browser and getting application ready]]>
            </line>
            <line>
              <![CDATA[Browser and application is up and running fine]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginApp()[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="loginApp" duration-ms="8550" started-at="2019-11-22T21:23:36Z" finished-at="2019-11-22T21:23:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="tearDownMethod" is-config="true" duration-ms="2" started-at="2019-11-22T21:23:45Z" finished-at="2019-11-22T21:23:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp status=SUCCESS method=LoginLogoutTest.loginApp()[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="logoutApp()[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="logoutApp" duration-ms="3243" started-at="2019-11-22T21:23:45Z" finished-at="2019-11-22T21:23:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutApp -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="tearDownMethod" is-config="true" duration-ms="2" started-at="2019-11-22T21:23:48Z" finished-at="2019-11-22T21:23:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=logoutApp status=SUCCESS method=LoginLogoutTest.logoutApp()[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.learnautomation.testCases.LoginLogoutTest@39c0f4a]" name="tearDown" is-config="true" duration-ms="1221" started-at="2019-11-22T21:23:48Z" finished-at="2019-11-22T21:23:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.learnautomation.testCases.LoginLogoutTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
